guide/web-scene.md

```diff
@@ -0,0 +1,71 @@
+## X3D and Web Scene
+
+### Description
+
+Webots can export a world to an interactive 3D `HTML` page.
+This feature is especially useful for publishing Webots-created worlds on the Web.
+
+When doing so, an [X3D](http://www.web3d.org/x3d/what-x3d) file and a `HTML5` page are generated.
+
+The `X3D` file is containing the graphical information of the world.
+`X3D` is a modern `XML`-based file format for representing 3D computer graphics.
+It can be imported in many 3D softwares (for visualizing, modeling, etc.).
```

Comment:
software is never plural.
If possible I would make a sentence instead of using brackets.